docs/concepts/macros/sqlmesh_macros.md

```diff
@@ -1681,6 +1681,73 @@ def some_macro(evaluator):
     ...
 ```
 
+#### Accessing model, physical table, and virtual layer view names
+
+All SQLMesh models have a name in their `MODEL` specification. We refer to that as the model's "unqualified" and "unresolved" name because it may not correspond to any specific object in the SQL engine.
```

Comment:
```suggestion
All SQLMesh models have a name in their `MODEL` specification. We refer to that as the model's "unresolved" name because it may not correspond to any specific object in the SQL engine.
```